scripts/articlesController.js

```diff
@@ -0,0 +1,9 @@
+var articlesController = {};
+
+articlesController.index = function() {
+  console.log('-> articlesController.index');
+  webDB.init();
```

Comment:
Your `webDB.init()` function does not currently set up SQL tables. Make sure that is done if needed at some point in your process.. If not, the first time your code runs on a new machine, you will run into an error.